src/main/java/knightminer/simplytea/data/gen/RecipeGenerator.java

```diff
@@ -108,9 +122,12 @@ protected void buildRecipes(Consumer<FinishedRecipe> consumer) {
 		addTeaWithBag(consumer, tea_leaf, teabag_green, cup_tea_green);
 		addTeaWithBag(consumer, black_tea, teabag_black, cup_tea_black);
 		addTeaWithBag(consumer, chorus_petal, teabag_chorus, cup_tea_chorus);
+		addTea(consumer, cup_water_hot, teapot_hot);
+		addTea(consumer, cup_frothed, teapot_frothed);
 
 		// advanced tea
 		addTea(consumer, cup_cocoa, Items.COCOA_BEANS, Items.COCOA_BEANS, teapot_frothed);
+		addTea(consumer, cup_cocoa, Items.COCOA_BEANS, cup_frothed);
```

Comment:
Why is this just 1 cocoa beans instead of 2? 1 cup milk + 2 cocoa = 1 cup cocoa.